tests/unit/fixtures/__init__.py

```diff
@@ -22,3 +27,9 @@ def basic_specs_fixture():
     _write('app', 'app-b', {'repo': 'github.com/app/b', 'image': 'app/b'})
     _write('lib', 'lib-a', {'repo': 'github.com/lib/a', 'image': 'lib/a'})
     _write('service', 'service-a', {'image': 'service/a'})
+
+def busybox_single_app_fixture(num_bundles=1):
```

Comment:
single_app_fixture implies this only does one, but it can do more.  Could clarify name